keepassxc-browser/options/options.css

```diff
@@ -227,8 +226,44 @@ table td:last-of-type {
     .sidebar {
         width: 240px;
     }
+    
+    main {
+      margin-left: 240px !important;
+      min-width: calc(0.8333333333 * 1440px) !important;
+      max-width: calc(100% - 240px) !important;
+      width: unset !important;
+    }
 
     footer {
         display: block;
     }
+    
+    .form-text, .form-group {
+        max-width: calc(0.8333333333 * 1440px);
+    }
+}
+
+
+#sitePreferencesTable > li > details > summary {
+    list-style: none;
+    vertical-align: middle;
+}
+
+#sitePreferencesTable > li > details > summary::-webkit-details-marker {
+    display: none;
+}
+
+#sitePreferencesTable > li > details > summary > div > div:first-child{
+    display: flex;
+    align-items: center;
+}
+
+#sitePreferencesTable > li > details > summary > div > div:first-child::before {
+    content: '►';
```

Comment:
If you want to go this way, you could just use `<details>` and `<summary>`. The About page is already using those standard HTML elements.

Reply:
The problem is that when I am using a list instead of a table, like I showed in the previous screenshots then the small triangle will move on top of the text as you can see [here](https://github.com/keepassxreboot/keepassxc-browser/pull/2358#issuecomment-2397034889). Thats why I did that, this way the delete button will wrap with the text if there is no space, instead of taking up half of the horizontal space the entire time. While it looks a bit more inconsistent I think this is better than the previously used horizontal scrolling as for me that felt like a very bad user experience.
![image](https://github.com/user-attachments/assets/8da748cc-1c3d-4fce-a238-b51d212de0fe)

Reply:
FYI: In my mockup I used `<details>` and `<summary>` only for the single cell.

Reply:
I agree that the delete button now possibly moving to the left is not optimal but I do not have enough bootstrap and flexbox knowledge to solve this problem, if you have any ideas I would be very happy to try those out. I think that going with the current list approach instead of a table would be better if the current problem could be solved. But if you think that having a table with a strict separation line is better than we could go back to a table.

Reply:
I would also want to see a simpler list implementation instead of an older table. It would be easier to manage from JavaScript too. I can try make some mockups later and see if I have the same issue.